data/json/effects.json

```diff
@@ -1730,6 +1737,11 @@
         "name": [ "Happy", "Happy", "Happy", "Happy", "Happy",
                   "Joyful", "Joyful", "Joyful", "Joyful", "Joyful",
                   "Elated" ],
+        "desc": [ "Life is good.", "Life is good.", "Life is good.", "Life is good.", "Life is good.",
+                  "Oh what a day!  What a lovely day!",
+                  "Oh what a day!  What a lovely day!", "Oh what a day!  What a lovely day!",
+                  "Oh what a day!  What a lovely day!", "Oh what a day!  What a lovely day!",
+                  "I am the King of the World, baby!." ],
```

Comment:
For the last line, to avoid any potential "gender" issues, it might just be easier to say `I'm on top of the world, baby!`

Reply:
I knew this would come up, I only regret that it kills the reference...